### ISSUE TITLE
imposible to update remote installations of RustDesk 1.2.0 nightly

### ISSUE THREAD

**Issue by mikekgr** [rustdesk/rustdesk#4595]
### Bug Description

Dear Sirs,
form long time ago I update my remote Windows 10/11 RustDesk installations using a local PowerShell script that is:

```
# don't forget to: Unblock the script
# Right-click the script and choose Properties. In the Properties dialog, if available, select the Unblock checkbox and press OK.
# inside the PowerShell run: Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
Write-Host "PowerShell script Version 0.3"
Write-Host "Downloading RustDesk installer / nightly build ..."
###$url = "https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.0-x86_64-pc-windows-msvc.exe"
$url = "https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.0-x86_64.exe"
###$output = "$env:USERPROFILE\Downloads\rustdesk-1.2.0-x86_64-pc-windows-msvc.exe"
$output = "$env:USERPROFILE\Downloads\rustdesk-1.2.0-x86_64.exe"
Invoke-WebRequest -Uri $url -OutFile $output -UseBasicParsing -ErrorAction SilentlyContinue
Write-Host "Download complete."

Write-Host "Running RustDesk nightly installer with update parameter..."
& $output --update
Write-Host "RustDesk installation / update complete."

Write-Host "Waiting for file to be closed..."
Write-Host "Press any key to exit, or the script will close automatically in 30 seconds..."
Start-Sleep -Seconds 25
Write-Host "Deleting downloaded file..."
Remove-Item $output -Force
Write-Host "Downloaded file deleted."

$timeout = New-TimeSpan -Seconds 30
$null = $Host.UI.RawUI.ReadKey('NoEcho,IncludeKeyDown', $timeout)
exit
```
`
From some days ago it is not possible to update the installations because the new way ask to accept the installation but I can not consent (it is turns to gray ...
The only way to do the update now is to connect to the remote PC using e.g. AnyDesk ,,,

Please guide me how to solve this big for me problem.

Thanks and Best Regards.
Mike Kranidis

### How to Reproduce

Just download the latest nightly build and run it, it will prompt you to update but you can't remotely ...

### Expected Behavior

To permit the update as it was the case till some days ago (6-7 days perhaps)

### Operating system(s) on local side and remote side

Windows 10 Tested with the same reported problem on Windows 11 as well

### RustDesk Version(s) on local side and remote side

Both running the today nightly release

### Screenshots

No at this time, if you need tell me to supply

**Comment by rustdesk** [comment]
Sorry, wontfix this for the time being before I can be convinced, old update is removed, because it has the other potential bugs, so I removed it to this one.

**Comment by mikekgr** [comment]
> Sorry, wontfix this for the time being before I can be convinced, old update is removed, because it has the other potential bugs, so I removed it to this one.

I don't try to install all updates. For example, I can't update my two days old nightly installation with the current, today nightly...
It is asking to consent BUT I CAN'T. The only workaround is to ask local user to consent or to connected using other remote program like AnyDesk. It is unsane to need AnyDesk in order to update RustDesk ...

**Comment by rustdesk** [comment]
use `--reinstall` or `--silent-install`, I will remove --update soon.

**Comment by rustdesk** [comment]
> use `--reinstall` or `--silent-install`, I will remove `--update` soon.

Actually  "--update" is already removed

**Comment by mikekgr** [comment]
> --silent-install

I just tried --reinstall to my remote (at home) PC and now I can not connect to at all. Leter when I will be back at home, I will see what is happened but seems that remotely  the update using --reinstall doesn't work

**Comment by rustdesk** [comment]
`--silent-install` should work, I will remove `--reinstall`

**Comment by mikekgr** [comment]
> `--silent-install` may work in your case, I will remove `--reinstall`

Yes I can confirm that --silent-install is working fine in my case, Thanks !
